cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+openapi: 3.1.0
+info:
+  version: 0.7.0
+  title: Common Card API (Public)
+  description: >
+    This specification defines a common card API for payment cards used in Switzerland.
+    The API is supposed to be used by third parties to retrieve payment card information for scheme-based debit, credit or prepaid cards (read only).
+  contact:
+    email: info@common-api.ch
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: https://card.common-api.ch
+externalDocs:
+  description: Find out more about SFTI API specifications
+  url: https://www.common-api.ch
+
+tags:
+  - name: Cards
+    description: Operations for retrieving information about payment cards.
+  - name: Card-tokens
+    description: Operations for retrieving information about tokens associated with payment cards.
+  - name: Transactions
+    description: Operations for retrieving information about transactions and authorizations done with payment cards or tokens.
+
+security:
+  - ApiKeyAuth: []
+  - OAuth2:
+      - read
+      - write
+
+paths:
+  /cards:
+    get:
```

Comment:
General: 
- Who can/may call this API?
- Use cases are missing: These are an important basis for the validation

Specific
- Are all of these values relevant?
- Which validations are carried out by whom?
- Which party delivers the answers? (Bank, PSP, or ???) --> in my opinion, it may only be the card issuer to avoid regulatory and compliance risks.
- How is it ensured that, for example an authority only sees his cards? In such a case, an IBAN alone is not sufficient for validation.
- More examples are mandatory – overall debit and credit; Viseca, SIX and, where applicable, SwissBankers for validation
- Contract reference: 
	• The field must be clearly defined. Is this a reference field of the issuer to the client? In the case of credit cards, this could be the credit card account; in the case of debit, the IBAN of a bank
	• The question is, however, if a client would like to query all of his cards of a bank, how should he do that?
- Card status: 
	• The functional status is 'active' and 'inactive' status. (temp.) 'Suspended' is the substatus. If 'Inactive', there can be the substatus 'blocked'. This needs to be resolved globally, not Viseca-specific. 
	• Is it correct and permissible to return cards that are no longer active?
Client with multiple accounts/cards: Does he/she need to send 1 API per request? Better is, that he can make one call and gets all cards by that issuer.

Reply:
In general please review which questions are still open for you after the answers for your comment on the operational guide.

Reply:
Comments will be used as validation when the first use case is implemented.